docs/changelog.rst

```diff
@@ -1,6 +1,16 @@
 Changelog
 ~~~~~~~~~
 
+0.5.0 (October 2022)
+-------------------
+
+* Updated iqm-client version requirement to 9.1.
+* IQMBackend initializer now does not accept a settings file.
+* IQMBackend initializer now retrieves device name and qubit names from backend
+server instead of configuration file.
+* IQMBackend initializer now retrieves qubit coupling (connectivity) and native
+gateset from backend server instead of hard-coded constants.
```

Comment:
```suggestion
* IQMBackend initializer now retrieves qubit coupling (connectivity) and native
  gateset from backend server instead of hard-coded constants.
```